internal/alb/lb/attributes.go

```diff
@@ -0,0 +1,204 @@
+package lb
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/elbv2"
+	"github.com/aws/aws-sdk-go/service/elbv2/elbv2iface"
+	"github.com/kubernetes-sigs/aws-alb-ingress-controller/internal/albctx"
+	"github.com/kubernetes-sigs/aws-alb-ingress-controller/pkg/util/log"
+	api "k8s.io/api/core/v1"
+)
+
+const (
+	DeletionProtectionEnabledString = "deletion_protection.enabled"
```

Comment:
Maybe like DeletionProtectionEnabledKey and DefaultDeletionProtectionEnabled ? 
but current workings for me too 😸 , just nitpicking...